examples/notebooks/rag/rag_1D_query_replicate.ipynb

```diff
@@ -249,33 +249,48 @@
     "\n",
     "### LLM Choices at Replicate\n",
     "\n",
-    "- llama 3.1 : Latest\n",
-    "    - **meta/meta-llama-3.1-405b-instruct** : Meta's flagship 405 billion parameter language model, fine-tuned for chat completions\n",
-    "- Base version of llama-3 from meta\n",
-    "    - [meta/meta-llama-3-8b](https://replicate.com/meta/meta-llama-3-8b) : Base version of Llama 3, an 8 billion parameter language model from Meta.\n",
-    "    - **meta/meta-llama-3-70b** : 70 billion\n",
-    "- Instruct versions of llama-3 from meta, fine tuned for chat completions\n",
-    "    - **meta/meta-llama-3-8b-instruct** : An 8 billion parameter language model from Meta, \n",
-    "    - **meta/meta-llama-3-70b-instruct** : 70 billion\n",
+    "\n",
+    "| Model                               | Publisher | Params | $ / 1M input | $ / 1M output | Description                                          |\n",
+    "|-------------------------------------|-----------|--------|--------------|---------------|------------------------------------------------------|\n",
+    "| ibm-granite/granite-3.0-8b-instruct | IBM       | 8 B    | $ 0.05       | $ 0.25        | IBM's newest Granite Model v3.0  (default)           |\n",
+    "| ibm-granite/granite-3.0-2b-instruct | IBM       | 2 B    | $ 0.03       | $ 0.25        | IBM's newest Granite Model v3.0                      |\n",
+    "| meta/meta-llama-3.1-405b-instruct   | Meta      | 405 B  | $ 9.5        | $ 9.5         | Meta's flagship 405 billion parameter language model |\n",
+    "| meta/meta-llama-3-8b-instruct       | Meta      | 8 B    | $ 0.05       | $ 0.05        |                                                      |\n",
+    "| meta/meta-llama-3-70b-instruct      | Meta      | 70 B   | $ 0.65       | $ 2.75        |                                                      |\n",
+    "\n",
+    "\n",
+    "(Prices are as of Oct 2024)\n",
     "\n",
```

Comment:
@sujee Since the prices keep changing all the time, even though you have commented that these are Oct. 24 prices, please remove the two price columns from this table (and the comment about Oct. 2024). Thanks.

Reply:
fixed model table @shahrokhDaijavad